watcher/src/bin/main.rs

```diff
@@ -30,9 +35,25 @@ fn main() {
         logger.clone(),
     )
     .expect("Could not create or open watcher db");
-    let watcher = Watcher::new(watcher_db, transactions_fetcher, logger);
-    // For now, ignore origin block, as it does not have a signature.
-    watcher
-        .sync_signatures(1, config.max_block_height)
-        .expect("Could not sync signatures");
+    let watcher = Watcher::new(watcher_db.clone(), transactions_fetcher, logger.clone());
+
+    let _verification_reports_collector = VerificationReportsCollector::new(
```

Comment:
For now we only collect verification reports inside the `mc-watcher` binary. `mc-mobilecoind` remains unchanged (it uses the `WatcherDB` and `WatcherSyncThread` to sync timestamps and block signatures but has no need for the verification reports).